### ISSUE TITLE
Attributes in CelebA pre-trained model

### ISSUE THREAD

**Issue by JosephPai** [yunjey/stargan#101]
Are all attributes in CelebA included in the pre-trained model?

Or just few of them? 

`'5_o_Clock_Shadow', 'Arched_Eyebrows', 'Attractive', 'Bags_Under_Eyes', 'Bald', 'Bangs', 'Big_Lips',
'Big_Nose', 'Black_Hair', 'Blond_Hair', 'Blurry', 'Brown_Hair', 'Bushy_Eyebrows', 'Chubby',
'Double_Chin', 'Eyeglasses', 'Goatee', 'Gray_Hair', 'Heavy_Makeup', 'High_Cheekbones', 'Male',
'Mouth_Slightly_Open', 'Mustache', 'Narrow_Eyes', 'No_Beard', 'Oval_Face', 'Pale_Skin', 'Pointy_Nose',
'Receding_Hairline', 'Rosy_Cheeks', 'Sideburns', 'Smiling', 'Straight_Hair', 'Wavy_Hair', 'Wearing_Earrings',
'Wearing_Hat', 'Wearing_Lipstick', 'Wearing_Necklace', 'Wearing_Necktie', 'Young'`

**Comment by rex-yue-wu** [comment]
No. The author replied in #97. Only the used 5 attributes are supported. You have to retrain the model yourself for other attributes.

**Comment by renmengyuan** [comment]
> No. The author replied in #97. Only the used 5 attributes are supported. You have to retrain the model yourself for other attributes.

Is there some idea about how to prepare the format of datasets if I only want to change the hairstyle?

**Comment by futurianh1k** [comment]
Dooray! 메일 발송 실패 안내



  
  
    
    
  
  
    
메일 발송
실패 안내
    
  
  
    
        
          
***@***.***) 
          님께
보낸
메일이
전송되지
못하였습니다.
        
        
          실패 사유를 확인해보세요.
        
    
  
  
    
      
        
          
            * 받는 사람 : 
***@***.***) 
          
          
            * 발송 시간 : 
2021-07-15T16:40:38
          
          
            * 메일 제목 : 
Re: [yunjey/stargan] Attributes in CelebA pre-trained model (#101)
          
          
            
              
                * 실패 사유 : 
              
              
                
받는 사람의 메일함 용량이 가득 찼습니다.
받는 사람에게 메일함 정리를 요청하신 후 다시 발송해 보세요.
                
              
            
                      
          
        
      
    
  
  
    
        
          이 메일은 발신전용으로 회신되지 않습니다.
          더 궁금하신 사항은
          ***@***.***
          으로 문의해 주시기 바랍니다.
        
    
  
  
    
        © Dooray!.

**Comment by kalikhademi** [comment]
if the pre-trained model includes attribute Young, can I change young ones to old and old one to youngs or only young to old is supported?
